src/interfaces/GameInstanceRequest.ts

```diff
@@ -0,0 +1,11 @@
+import { Platforms } from "../constants/Platforms"
+
+export default interface GameInstanceRequest {
+    rating?: number;
+    review?: string;
+    ranking?: string;
+    yearPlayed?: number;
+    yearReceived?: number;
+    notes?: string;
+    platformsOwnedOn?: Array<string>; // TODO: will want to change to Platform enum
```

Comment:
Any reason we can't do this now?

Reply:
it's been a while since I've done this tbh, but I think it had something to do with the fact it's not implemented in the backend yet

Reply:
Ah we do have this https://github.com/trauman413/gaming-backlog-api-kotlin/blob/development/src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/enums/Platforms.kt does that work?

Reply:
We have the enums defined there, but I think because Platforms in the Game/GameInstance model is a String (https://github.com/trauman413/gaming-backlog-api-kotlin/blob/development/src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/Game.kt#L14) it'll break if I just change it in the frontend. It'll probably be fast to change them both at some point

Reply:
Ahh okay gotcha, mind filing a ticket? Then this should be good to go

Reply:
https://gaming-backlog.atlassian.net/browse/GB-118